snapcraft/_store.py

```diff
@@ -550,6 +619,13 @@ def history(snap_name, series, arch):
     print(tabulated_revisions)
 
 
+def get_latest_revision(snap_name, series=None, arch=None):
+    store = storeapi.StoreClient()
+    with _requires_login():
+        history = store.get_snap_history(snap_name, series, arch)
+    return history[-1]
```

Comment:
it will be useful once we download something to cache before doing a delta or I might take this for the "download and unpack the core snap" work to make classic buildable. But it should leave this PR as it will be forgotten :-)